Bonsai.Tinkerforge/DeviceData.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bonsai.Tinkerforge
+{
+    /// <summary>
+    /// Data representation of a connected TinkerForge module
+    /// </summary>
+    public class DeviceData
```

Comment:
Since we don't seem to be serializing this anymore, can we make this class internal?

Reply:
Also we could move this class definition inside the `UidConverter` class and rename this file as either `TinkerforgeDeviceLookup.cs` or maybe just move both of these types inside `UidConverter`, as they seem to be implementation details which are only used there anyway?